app/src/main/java/com/randomappsinc/studentpicker/home/NameListsAdapter.java

```diff
@@ -117,19 +129,23 @@ public void onClick(@NonNull MaterialDialog dialog, @NonNull DialogAction which)
                         preferencesManager.removeNameList(getItem(position));
                         content.remove(position);
                         setNoContent();
-                        notifyDataSetChanged();
+                        notifyItemRemoved(position);
+                        notifyItemRangeChanged(position, 1);
```

Comment:
What kind of bug happened if we didn't have this line again? The UI wouldn't update with the removed row?

Reply:
Yes, the position will not change I will make a video of the bug.